stone/target/python_rsrc/stone_serializers.py

```diff
@@ -24,11 +24,471 @@
 except (SystemError, ValueError):
     # Catch errors raised when importing a relative module when not in a package.
     # This makes testing this file directly (outside of a package) easier.
-    import stone_validators as bv  # type: ignore
+    import stone_validators as bv # type: ignore
```

Comment:
End of line comments are separated from the code by two spaces: https://www.python.org/dev/peps/pep-0008/#inline-comments